test/js/base/struct.js

```diff
@@ -2,6 +2,8 @@ const assert = require("assert");
 const config = require("../config");
 const ripe = require("../../../src/js");
 
+const { Blob } = require("buffer");

```

Comment:
If there are collisions we can rename the variables to _buffer or buffer_ or if the number of places to edit are too many we can import this as `bufferNode`

Reply:
yes, we have some exceptions (eg: tobias-bot and slack bot mostly because the examples of equivalent bots use this standard) but when using common js lets import the complete object instead of unpacking its values